pkg/scheduler/core/generic_scheduler.go

```diff
@@ -451,7 +456,7 @@ func podFitsOnNode(
 		// TODO(bsalamat): consider using eCache and adding proper eCache invalidations
 		// when pods are nominated or their nominations change.
 		eCacheAvailable = eCacheAvailable && !podsAdded
-		for _, predicateKey := range predicates.PredicatesOrdering() {
+		for _, predicateKey := range predicates.Ordering() {
```

Comment:
Why change this?

Reply:
Because `PredicatesOrdering` caused the error:
`pkg/scheduler/algorithm/predicates/predicates.go:152:6: func name will be used as predicates.PredicatesOrdering by other packages, and that stutters; consider calling this Ordering`

Reply:
Do we touch any code about `PredicatesOrdering` ? Not sure why it works before but failed in this PR :(.

Reply:
Found the code about renaming.